Titta.m

```diff
@@ -519,7 +526,7 @@ function setOptions(obj,settings)
             else
                 wfunc = @pause;
             end
-            while true
+            while isempty(address)
```

Comment:
Hmm, i don't think all this logic can be skipped. We need to check the provided eye tracker address connects to an eye tracker of the model set in the settings. also, this `while isempty(address)` is opaque, even if it would work correctly. I'll be scratching my head later.
To do this right, i need to add something to the mex file. After you fix the other things i'll merge the pull request and then i'll fix up the logic here.

Reply:
Right, it would be better to be more careful here 👍🏾